### ISSUE TITLE
[$1000] Password - Infinite loading displays after login with changed password

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#9933]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Login to the app
2. Navigate to Settings > Security > Change password
3. Enter correct information in the 3 fields and verify you’re able to save the changes
4. 3 Log out of the app
5. Login with the new Password

## Expected Result:
User is able to log in with new password

## Actual Result:
Infinite loading displays after login with changed password

## Workaround:
Unknown

## Platform:

Where is this issue occurring?

- Web
- iOS
- Android
- Desktop App
- Mobile Web

**Version Number:** 1.1.84.11

**Reproducible in staging?:** Yes

**Reproducible in production?:** Yes

**Email or phone of affected tester (no customers):** any

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/93399543/179100398-89c67e20-d504-443e-ae83-58b4a77aeba1.mp4

**Expensify/Expensify Issue URL:** 

**Issue reported by:** Applause - Internal Team

**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by flodnv** [comment]
@kbecciv I am unable to reproduce on desktop web in chrome. Perhaps this is a mobile only bug?

**Comment by kbecciv** [comment]
@flodnv This issue is reproduced in all environments, only we did not check the Desktop. Please attached the videos.

https://user-images.githubusercontent.com/93399543/180809828-e8fb23d9-35d1-4b39-83ca-e56d27004cef.mp4


https://user-images.githubusercontent.com/93399543/180809862-2036b9d9-7204-4c60-beca-55ef15b1283b.mp4


https://user-images.githubusercontent.com/93399543/180809921-dad74500-36b6-42d1-ab0d-7648a4657870.mp4


https://user-images.githubusercontent.com/93399543/180809934-eacf7543-cef4-461e-b71c-81d68b7bdabf.mp4


https://user-images.githubusercontent.com/93399543/180809943-2436fe06-051d-4b5a-a6de-81b1b00b2592.mp4

**Comment by flodnv** [comment]
Thanks 👍

**Comment by stephanieelliott** [comment]
Posted to Upwork: https://www.upwork.com/jobs/~019540b6b8ab24a555

**Comment by stephanieelliott** [comment]
2x'd in Upwork to $500.

**Comment by stephanieelliott** [comment]
@flodnv I think we typically keep these weekly, is there a reason why this one was changed to monthly?

**Comment by stephanieelliott** [comment]
2x'd in Upwork to $1000

**Comment by vladnobenladen** [comment]
This issue is no longer reproducible on latest `main`.
After hard research and code review, I found that this issue is already fixed by @luacmartins's [PR](https://github.com/Expensify/App/pull/10051) which was merged on Aug 10.
Build v1.1.88-7 was generated on Aug 8 and build v1.1.88-8 was generated on Aug 15.
Therefore, the issue is gone from v1.1.88-8, but the reporter tested on v1.1.84-11 (generated on Jul 13).
